src/UKHO.ERPFacade.PublishPriceChange.WebJob/Program.cs

```diff
@@ -97,7 +97,7 @@ private static void ConfigureServices(IServiceCollection serviceCollection, ICon
             //create the logger and setup of sinks, filters and properties	
             Log.Logger = new LoggerConfiguration()
                 .MinimumLevel.Information()
-                .WriteTo.File("Logs/UKHO.ERPFacade.WebJob-.txt", rollingInterval: RollingInterval.Day, outputTemplate: "{Timestamp:yyyy-MM-dd HH:mm:ss} [{Level}] [{SourceContext}] {Message}{NewLine}{Exception}")
+                .WriteTo.File("Logs/UKHO.BulkPrice.WebJob-.txt", rollingInterval: RollingInterval.Day, outputTemplate: "{Timestamp:yyyy-MM-dd HH:mm:ss} [{Level}] [{SourceContext}] {Message}{NewLine}{Exception}")
```

Comment:
Can this be renamed as PublishPriceChange rather than BulkPrice

Reply:
Renamed to PublishPriceChange